openedx/features/qverse_features/registration/tests/test_register.py

```diff
@@ -0,0 +1,78 @@
+"""
+Tests for account creation.
+"""
+from ddt import ddt, data
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.test.utils import override_settings
+from django.urls import reverse
+
+from openedx.features.qverse_features.registration.models import (QVerseUserProfile,
+                                                                  MAX_LEVEL_CHOICES,
+                                                                  MAX_PROGRAMME_CHOICES)
+from openedx.features.qverse_features.registration.tests.factories import DepartmentFactory
+
+from student.models import UserProfile
+
+
+@ddt
+@override_settings(
+    REGISTRATION_EXTENSION_FORM='openedx.features.qverse_features.registration.forms.QVerseUserProfileForm',
+)
+class CreateAccountTests(TestCase):
+    """
+    Tests for account creation.
+    """
+    def setUp(self):
+        super(CreateAccountTests, self).setUp()
+        settings.FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION'] = True
```

Comment:
Kindly use @overridesettings decorator here, otherwise, it will change this value in every test

Reply:
I tried
```
@override_settings
```
but it overrides complete features dictionary which is not the required behavior here.

Reply:
You can create a copy of FEATURES and then update that copy and assign it.
settings.FEATURES.copy().update(
YOUR_FLAG
)
It may seem tricky, but its necessary otherwise all the tests in opened will have the FLAG overiden.

Reply:
done.